go/interpreter/lfvm/ct.go

```diff
@@ -209,7 +209,9 @@ func convertCtMemoryToLfvmMemory(memory *st.Memory) *Memory {
 
 	result := NewMemory()
 	size := uint64(len(data))
-	_ = result.expandMemoryWithoutCharging(size)
+	value, validSize, _ := result.getExpansionCostsAndSize(size)
+	result.store = make([]byte, validSize)
+	result.currentMemoryCost = value
```

Comment:
I like this, but I kind of want to add a comment explicitly saying this is done like this because it's an internal conversion from LFVM to CT, and it should not be done anywhere else. Or some kind of warning. Then again, this holds for the whole file. So maybe not necessary

Reply:
the code has been refactored to call mem.set directly.